modules/creating-new-osdk-project.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * operators/migrating-to-osdk-v0.1.0.adoc
+
+[id='creating-new-operator-sdk-project_{context}']
+= Creating a new Operator SDK project
+
+Rename your v0.0.x project and create a new v0.1.0 project in its place.
+
+. Ensure the SDK version is v0.1.0:
++
+----
+$ operator-sdk --version
+operator-sdk version 0.1.0
+----
+
+. Create new project:
++
+----
+$ cd $GOPATH/src/github.com/example-inc/
+$ mv memcached-operator old-memcached-operator
+$ operator-sdk new memcached-operator
```

Comment:
Nit: We can add the flag `skip-git-init` here since we're copying over the `.git` directory from the old project:
```sh
$ operator-sdk new memcached-operator --skip-git-init
```

Just updated this in our migration guide as well.
https://github.com/operator-framework/operator-sdk/pull/688